x-pack/test/functional/apps/saved_objects_management/exports/_6.8_import_saved_objects.json

```diff
@@ -0,0 +1,642 @@
+[
```

Comment:
@bhavyarm do we need this file in master? I thought that you will replace `_7.12_import_saved_objects.ndjson` with it in 7.x branch

Reply:
@dmlemeshko right. Thank you. I will delete it from 8.0.0. I checked in both the files instead of doing only one.

Reply:
Done @dmlemeshko Thank you!